sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -430,6 +432,22 @@ public partial class ServiceBusSessionReceiverOptions
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override string ToString() { throw null; }
     }
+    public sealed partial class ServiceBusSharedAccessKeyCredential
+    {
+        public ServiceBusSharedAccessKeyCredential(string sharedAccessSignature) { }
+        public ServiceBusSharedAccessKeyCredential(string sharedAccessKeyName, string sharedAccessKey) { }
+        public string SharedAccessKey { get { throw null; } }
+        public string SharedAccessKeyName { get { throw null; } }
+        public string SharedAccessSignature { get { throw null; } }
```

Comment:
Why are the Key and Signature public?  We should keep them internal.

Reply:
Fair point.. 
@jsquire Other than giving the user the ability to check what the current value of their key/signature is if they were ever rotating them, is there any other reason to have these public? Is there a security angle here where we would not want to make them public to avoid the case of this info getting logged accidentally?

Reply:
The intent is to allow inspection by callers holding the token, as context for making the decision of "is this using the latest information or do I need to rotate" as well as debugging authorization failures by understanding the same context.

What would the benefit of hiding this information from developers using the library?

Reply:
We've done this for other credentials because although it's not a security issue, we want customers to be thoughtful about how they manage their secrets and we make it inconvenient to read this info from our types.

Reply:
I don't agree with the rationale, but I don't feel strongly enough to push back on it.  I'm comfortable deferring to @ramya-rao-a , assuming that @tg-msft has cleared this with @KrzysztofCwalina.